src/main/java/com/google/devtools/build/lib/exec/ExecutionOptions.java

```diff
@@ -105,7 +102,19 @@
       help =
           "Specify how to distribute compilation of other spawn actions. "
               + "Example: 'Javac=local' means to spawn Java compilation locally. "
-              + "'JavaIjar=sandboxed' means to spawn Java Ijar actions in a sandbox. ")
+              + "'JavaIjar=sandboxed' means to spawn Java Ijar actions in a sandbox. "
+              + "You can pass a comma-separated list of strategies, "
+              + "for example: --spawn_strategy=remote,worker,linux-sandbox. "
+              + "For any action that it wants to execute, Bazel just picks the first strategy from the given list "
+              + "that can execute the action. "
+              + "If none of the strategy flags was used, bazel will generate a default list of strategies "
+              + "'remote,worker,sandboxed,local' and, for every action it wants to execute, "
+              + "will pick up the first strategy that can execute it. "
+              + "See https://blog.bazel.build/2019/06/19/list-strategy.html for details. \n"
+              + "'remote' meant run all of them remotely"
+              + "'standalone' or 'local' means run all of them locally without any kind of sandboxing. "
+              + "'sandboxed' means to run them in a sandboxed environment with limited privileges "
+              + "(details depend on platform support).")
```

Comment:
That text might be a little long for a help text. Consider changing it to 

`Accepts a comma-separated list of strategies from highest to lowest priority. For each action Bazel picks the strategy with the highest priority that can execute the action. The default value is "remote,worker,sandboxed,local". See https://blog.bazel.build/2019/06/19/list-strategy.html.`

Reply:
done for both